libs/methodologies/bold/rule-processors/credits/src/rewards-distribution/rewards-distribution.test-cases.ts

```diff
@@ -0,0 +1,637 @@
+    ...createHaulerRewards(),
+    createMassIdActorReward(
+      WASTE_GENERATOR,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[WASTE_GENERATOR],
+    ),
+    createMassIdActorReward(
+      PROCESSOR,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[PROCESSOR],
+    ),
+    createMassIdActorReward(
+      RECYCLER,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[RECYCLER],
+    ),
+    createMassIdActorReward(
+      INTEGRATOR,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[INTEGRATOR],
+    ),
+    createMassIdActorReward(
+      APPOINTED_NGO,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[APPOINTED_NGO],
+    ),
+    createMethodologyActorReward(
+      METHODOLOGY_AUTHOR,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[METHODOLOGY_AUTHOR],
+    ),
+    createMethodologyActorReward(
+      METHODOLOGY_DEVELOPER,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[METHODOLOGY_DEVELOPER],
+    ),
+    createMethodologyActorReward(
+      NETWORK,
+      MULTI_HAULER_REWARDS_DISTRIBUTION[NETWORK],
+    ),
+  ] as unknown as NonEmptyArray<MassIdReward>;
+
+  return {
+    massIdDocumentId: documentId,
+    massIdRewards,
+  };
+};
+
+const buildCertificateDocuments = (options: {
+  documentId: string;
+  rewardsDistribution: CertificateRewardDistributionOutput;
+  value: number;
+}) => {
+  const { documentId, rewardsDistribution, value } = options;
+
+  return new BoldStubsBuilder({
+    massIdActorParticipants,
+    massIdDocumentIds: [documentId],
+  })
+    .createMassIdDocuments({
+      partialDocument: {
+        subtype: FOOD_FOOD_WASTE_AND_BEVERAGES,
+      },
+    })
+    .createMassIdAuditDocuments()
+    .createCertificateDocuments({
+      externalEventsMap: {
+        [RULES_METADATA]: stubBoldCertificateRulesMetadataEvent({
+          metadataAttributes: [
+            [
+              REWARDS_DISTRIBUTION_RULE_RESULT_CONTENT,
+              rewardsDistribution as AnyObject,
+            ],
+          ],
+        }),
+      },
+      partialDocument: {
+        currentValue: value,
+        type: RECYCLED_ID,
+      },
+    })
+    .createCreditsDocument({
+      externalEventsMap: {
+        [RULES_METADATA]: stubBoldCreditsRulesMetadataEvent({
+          metadataAttributes: [[UNIT_PRICE, UNIT_PRICE_VALUE]],
+        }),
+      },
+    })
+    .build();
+};
+
+const createTestDocuments = () => {
+  const massId1Value = 1000;
+  const massId1DocumentId = faker.string.uuid();
+
+  const standardDocuments = buildCertificateDocuments({
+    documentId: massId1DocumentId,
+    rewardsDistribution: createStandardRewardsDistribution(massId1DocumentId),
+    value: massId1Value,
+  });
+
+  const multiHaulerDocuments = buildCertificateDocuments({
+    documentId: massId1DocumentId,
+    rewardsDistribution:
+      createMultiHaulerRewardsDistribution(massId1DocumentId),
+    value: massId1Value,
+  });
+
+  const massId2Value = 500;
+  const massId2DocumentId = faker.string.uuid();
+  const secondDocuments = buildCertificateDocuments({
+    documentId: massId2DocumentId,
```

Comment:
This same change should be applied to similar places.

```suggestion
    documentId: massIdCertificate2DocumentId,
```

Reply:
It is the same logic as the others.